sample/src/androidTest/java/com/uber/autodispose/TestKotlinActivity.kt

```diff
@@ -31,7 +33,7 @@ import java.util.concurrent.TimeUnit
  * Test Activity class to verify compilation of various extension functions.
  */
 @Ignore("Since it's only used to verify compilation of the extension functions")
-class TestKotlinActivity : AppCompatActivity() {
+class TestKotlinActivity : AppCompatActivity(), ScopeProvider {
```

Comment:
Just to doubly confirm - it doesn't get confused about ambiguous types here if targeting this now?

Reply:
No, it doesn't. Basically, it will always just resolve to one of those extension functions. It will ask you for one of them. If you're overloading with the `untilEvent`, it will always resolve to the one with `LifecycleOwner`.